application/src/main/java/uk/nhs/hee/tis/revalidation/service/RecommendationServiceImpl.java

```diff
@@ -273,6 +274,9 @@ else if(outcome.equals(UNDER_REVIEW.getOutcome())) {
     checkRecommendationStatus(gmcNumber, doctorsForDB.getDesignatedBodyCode());
     log.info("Fetching snapshot record for GmcId: {}", gmcNumber);
 
+    doctorsForDB.setDoctorStatus(getRecommendationStatusForTrainee(gmcNumber));
+    doctorsForDBRepository.save(doctorsForDB);
+
```

Comment:
It looks like the log statement above this relates to the code below.  I'd consider moving this above as it is not.